vowpalwabbit/action_score.h

```diff
@@ -1,79 +1,102 @@
 #pragma once
 namespace ACTION_SCORE
 {
-
 struct action_score
-{ uint32_t action;
+{
+  uint32_t action;
   float score;
 };
 
 typedef v_array<action_score> action_scores;
 
-class score_iterator :
-  public virtual std::iterator<
-	  std::random_access_iterator_tag,	// iterator_cateogry
-	  float,					// value_type
-	  long,						// difference_type
-	  float*,					// pointer
-	  float						// reference
-	  >
+class score_iterator : public virtual std::iterator<std::random_access_iterator_tag,  // iterator_cateogry
+                                                    float,                            // value_type
```

Comment:
I do not think we want this kind of alignment, can we change the option to just indent these one level instead of align

Reply:
I think this is now partially fixed - it depends on the interpretation of one level of indentation: https://github.com/VowpalWabbit/vowpal_wabbit/pull/1701/files#diff-2f6bec9b76c178b3a45f8d064aca5350L12